packages/pyroscope-datasource-plugin/grafana-provisioning/dashboards/with-dynamic-variable.json

```diff
@@ -77,7 +77,7 @@
         "multi": false,
         "name": "appName",
         "options": [],
-        "query": ".*",
+        "query": "apps()",
```

Comment:
This changes makes it so that variable queries like `.*` will stop working. I'm not sure if we need to maintain the backward compatibility

@eh-am what do you think?

Reply:
Nah go for it, this change hasn't even been released yet